assets-src/js/main/account-menu.js

```diff
@@ -54,7 +55,8 @@ let accountMenu = function () {
 		toggleButton.innerHTML = '<span class="sr-only">' + translate.translate('my-account', languageCode) + ' <span class="visuallyhidden">(' + translate.translate( 'logged-in' , languageCode) + ')</span></span><div class="avatar avatar--small icon"><img alt="" src="' + userInfo.avatar.thumbnail + '"/></div>';
 
 		// Media query event handler
-		let mq = window.matchMedia('(min-width: 71.25em)');
+		let mqValue = maxWidth / 16;
+		let mq = window.matchMedia('(min-width: ' + mqValue + 'em)');
```

Comment:
Why not go with `70em` directly?

Reply:
I think this makes it clearer where the value is coming from, and it follows the approach used in the script for the global navigation. But I can remove it and just go for the value, if you prefer?

Reply:
Thanks for the explanation. Let's leave it like that.